code/game/machinery/computer/ai_core.dm

```diff
@@ -26,7 +26,6 @@
 				var/obj/item/weldingtool/WT = P
 				if(!WT.isOn())
 					to_chat(user, "The welder must be on for this task.")
-					return
 				playsound(loc, 'sound/items/welder.ogg', 50, 1)
 				if(do_after(user, 20/P.toolspeed))
 					if(!src || !WT.remove_fuel(0, user)) return
```

Comment:
return TRUE?

also a few empty returns below that should probably be return TRUE since they're below istypes